farsetlabs.json

```diff
@@ -0,0 +1,38 @@
+{
+    "api": "0.13",
+    "botname": "FarBot",
+    "spacefed": {
+        "spacenet": false,
+        "spacesaml": false,
+        "spacephone": false
+    },
+    "contact": {
+        "email": "admin@farsetlabs.org.uk",
+        "github":"https://github.com/FarsetLabs",
+        "twitter": "@farsetlabs",
+        "facebook": "https://www.facebook.com/FarsetLabs",
+        "slack": "http://slackin.farsetlabs.org.uk/",
+        "flickr": "http://www.flickr.com/groups/farset_labs",
+        "instagram":"https://www.instagram.com/farsetlabs",
+        "irc": "irc://irc.freenode.net/#farsetlabs",
+        "phone": "+448458694619",
+        "discourse":"https://discourse.farsetlabs.org.uk"
+    },
+    "issue_report_channels": [
+        "email",
+        "twitter"
+    ],
```

Comment:
```suggestion
    "issue_report_channels": [
        "email",
        "slack"
    ],
```

Reply:
when you test the file against https://spaceapi.io/validator/ it comes back with this error

{"valid":false,"message":"(root).issue_report_channels.1: issue_report_channels.1 must be one of the following: \"email\", \"issue_mail\", \"twitter\", \"ml\"\n"}

Reply:
Ah, thanks. I didn't know about that.

Reply:
```suggestion
    "issue_report_channels": [
        "email"
    ],
```